packages/babel-preset/esm.js

```diff
@@ -0,0 +1,49 @@
+const envPreset = require('@babel/preset-env');
+const modulesPreset = require('@babel/preset-modules');
+const reactPreset = require('@babel/preset-react');
+const plugins = require('./plugins');
+
+function preset(api, options = {}) {
+  const {
+    runtime,
+    useBuiltIns,
+    debug,
+    loose = true,
+    modules = api.env() === 'test',
+    development = api.env() === 'test',
+  } = options;
+
+  const presets = [
+    [modulesPreset, { loose }],
+
+    useBuiltIns && [
+      envPreset,
+      {
+        debug,
+        loose,
+        targets: { esmodules: true },
+        exclude: [/(transform|proposal)/],
+        modules: false,
+        shippedProposals: true,
+        useBuiltIns: 'usage',
+        corejs: 3,
+      },
+    ],
+    [reactPreset, { development }],
+  ].filter(Boolean);
+
+  return {
+    presets,
+    plugins: [
+      !!modules && require.resolve('@babel/plugin-transform-modules-commonjs'),
+      ...plugins({
+        loose,
+        modules,
+        runtime,
+        corejs: false,
```

Comment:
we don't want corejs in runtime right? libraries should require users polyfill as needed

Reply:
then wouldn't the right thing to do be to assert that the target is a library target rather than an app target?

Reply:
also, wouldn't the idea be to build these bundles to target newer browsers anyway?

Reply:
this preset doesn't support "target" right?

Reply:
so... i _feel_ like libraries shouldn't generally require applications to bring their own polyfills in general. idk.

idk, like... if someone is using preset-env to polyfill and only polyfilling their own code (because they're not running babel on node_modules), then... stuff just breaks, no?

Reply:
I think the common pattern is that libraries can require polyfills for older browser support, like React does with Set, Map, etc. For bleeding edge things i think those should really be explicitly included by the library